src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -10667,13 +10667,15 @@ protected virtual bool PrintMembers(System.Text.StringBuilder builder)
                 Row(21, TableIndex.TypeRef, EditAndContinueOperation.Default),
                 Row(4, TableIndex.TypeSpec, EditAndContinueOperation.Default),
                 Row(3, TableIndex.StandAloneSig, EditAndContinueOperation.Default),
-                Row(10, TableIndex.MethodDef, EditAndContinueOperation.Default)); // R.PrintMembers
+                Row(10, TableIndex.MethodDef, EditAndContinueOperation.Default), // R.PrintMembers
+                Row(24, TableIndex.CustomAttribute, EditAndContinueOperation.Default));
```

Comment:
What is this attribute?

Reply:
By looking at what we emit, it is the NullableContextAttribute. This matches my expectation because the record-structs branch fixed nullability annotations on the synthesized PrintMembers.

But looking at the metadata reader (`reader1`) something looked off (index out of bounds). Talked to @davidwengier, but he's not sure what to expect (this may be the result of EnC doing row mappings...). Will check with @tmat

```
            EditAndContinueLogEntry temp = reader1.GetEditAndContinueLogEntries().Last();
            CustomAttribute temp2 = reader1.GetCustomAttribute((CustomAttributeHandle)temp.Handle);
```

Reply:
Confirmed with Tomas that trying to get the attribute with `reader1.GetCustomAttribute(...)` is expected to fail.
He suggested that I add `#nullable enable` to the test to revert to previous behavior. Currently attribute changes are not considered supported by EnC. 
Filed https://github.com/dotnet/roslyn/issues/52786